jina/peapods/peas/__init__.py

```diff
@@ -404,11 +407,3 @@ def role(self) -> 'PeaRoleType':
 
         .. #noqa: DAR201"""
         return self.args.pea_role
-
-    @property
```

Comment:
Just for my understanding, what's the rationale for removing the `_is_inner_pea` property?

Reply:
that nobody is using and is private

Reply:
understood, thank you